deployments/cdi/README.md

```diff
@@ -0,0 +1,13 @@
+# SR-IOV Network Device Plugin in CDI mode deployment
+
+SR-IOV Network Device Plugin supports [Container Device Interface (CDI)](https://github.com/container-orchestrated-devices/container-device-interface).
+
+To enable CDI mode, SR-IOV Network Device Plugin should be started with `--use-cdi` CLI argument.
+This mode has different deployment requirements: `sriovdp-daemonset.yaml`
+
+```yaml
+    - mountPath: /etc/cdi/
+      name: default-cdi
+    - mountPath: /var/run/cdi
```

Comment:
we need just /var/run/cdi imo we dont deal with statically configured files right ?

Reply:
we should align daemonsets as well